### ISSUE TITLE
start attribute not working in toolbar

### ISSUE THREAD

**Issue by jgw96** [driftyco/ionic#6844]
#### Short description of the problem:
Putting the `start` attribute on `<ion-buttons>` inside an `<ion-toolbar>` does not position the `<ion-buttons>` element correctly. 

#### What behavior are you expecting?
The start attribute to position the buttons at the "front" of a toolbar

**Which Ionic Version?** 2.0.0-beta.8

http://plnkr.co/edit/me3Uk0GKWVRhZWU0usad?p=preview

Related to https://github.com/driftyco/ionic/issues/6842

**Comment by Luchillo** [comment]
I see this is rather simple, the end attribute work using an order property of 5, normal `ion-buttons` have order 4, the toolbar content has order 3, so it is expected that the start attribute set the order to <= 2, however the start attribute style is inexistent, this is very easy to fix.

**Comment by Luchillo** [comment]
Fix is as easy as this, however idk nor have enough time to make a pull:

```scss
ion-buttons[start] {
  order: 2;
}
```

**Comment by jgw96** [comment]
Hello. So after talking to @brandyscarney about this , this is actually correct behavior. [Here](http://ionicframework.com/docs/v2/api/components/toolbar/Toolbar/#buttons-in-a-toolbar) is a link to the docs on this.

**Comment by Luchillo** [comment]
Really? I know it's how material design works but since i'm using the toolbar to replace the modals navbar until fixed, i need to put it to the left to be consistent in the navigation with arrow back icon.

I was expecting that `start` property would stick it to the left of content, guess i'll stick with this hack until the navbar in modals get fixed.

**Comment by brandyscarney** [comment]
```
<ion-toolbar position="bottom">
  <ion-buttons start>Button</ion-buttons>
  <ion-title>Title</ion-title>
</ion-toolbar>
```

`start` places it to the left of the title in `ios` and to the right of the title in `md` and `wp` mode. If you'd like a button to always be on the left, regardless of the mode, use the `left` property instead. The plunker isn't working for me, but I am seeing these attributes working properly.

**Comment by Fthi-S3074641** [comment]
I replaced `start` by `left` and `end` by `right`

Now, I get a consistent alignment on both `ios` and `Android`.
Cheers!

**Comment by goleary** [comment]
Curious why start is to the left in ios & right in MD?  Not super intuitive, thought something was broken for quite some time.  Haven't seen this paradigm in other android app's i've used

**Comment by brandyscarney** [comment]
@goleary It used to be the case where iOS apps would have buttons on both the left and right of a title, and Material Design would always place them on the right of the title. The Twitter app was a good example of this:

<img src="http://img.deusm.com/informationweek/2013/12/1113033/twitterupdates1.jpg">

We wanted to make it easy for users to write the code once and have the buttons switch sides based on the mode. This isn't really the case anymore. We've changed this in the latest version so that `start` and `end` will mean `left` and `right`, respectively, in LTR mode (and opposite in RTL).

Right now we have `mode-start` and `mode-end` as the "new" way to do this, but I'm thinking of removing it completely. 🤔

Here is where we've documented these changes for v4: https://github.com/ionic-team/ionic/blob/core/BREAKING.md#attributes-renamed-3
